NOTICE

```diff
@@ -4,11 +4,19 @@ Copyright 2024 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
-This product includes software licensed under the terms
-of the following licenses, see LICENSE file:
-* Apache Software License, Version 2.0
-* Creative Commons 1.0 Universal
-* Eclipse Distribution License, Version 1.0
-* Eclipse Public License, Version 1.0
-* Eclipse Public License, Version 2.0
-* GNU General Public License, Version 2 with the GNU Classpath Exception
+The initial code for the Polaris project was donated
+to the ASF by Snowflake Inc. (https://www.snowflake.com/) copyright 2024.
+
+Apache Iceberg
+Copyright 2017-2022 The Apache Software Foundation
```

Comment:
Do we really need the year(s) in the file?
Why not `and onwards` here as well?

Reply:
We don't have the choice: we have to include the NOTICE file content as it is from ALv2 code used. So it's directly the content of Iceberg NOTICE file.